### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by tolga47** [speconsult/cordova-plugin-usagestats#1]
How we can use this plugin?
Do you have an example?

**Comment by spec2e** [comment]
Hi there 
Yes, however this plugin was only created as some test to obtain data from Android. 
I have updated the readme.md, so you can check it out :-)

Cheers
Søren

**Comment by tolga47** [comment]
Thankyou so mush :)
And can we see another apps usage statics with this plugin?
With package name.

**Comment by spec2e** [comment]
Yes. You can see all the apps that has been running, and are exposed using
the UsageStatsManager in Android

Den 6. dec. 2016 9.30 AM skrev "tolga47" <notifications@github.com>:

**Comment by tolga47** [comment]
Thanx but i can use plugin :(
I trying in cordova.
First i installed the plugin:
cordova plugin add https://github.com/spec2e/cordova-plugin-usagestats

Then i used js code in index.js
window.usageStats.usage('com.tf.indirkazan', function(e){ console.log(e); }, null);

**Comment by spec2e** [comment]
Try to clone this repo: 
https://github.com/spec2e/whitelist
It should work, if you deploy it on an Android unit. Just comment out the services that send statistics to a web service.

**Comment by tolga47** [comment]
window.usageStats.usage('com.tf.indirkazan', function(e){ console.log(e); }, null);

Is that correct using?

**Comment by tolga47** [comment]
Do you have an example for cordova :)

**Comment by spec2e** [comment]
It uses cordova. Ionic framework is a layer on top of cordova. What you see in the repo I provided, would be usable in any javascript/cordova application.
You can call the plugin with any argument, as long as you provide a success and error callback.
It does not do anything with the first argument (name). As I said, it was just something I created to try it out. Try something like:

```javascript
usageStats.usage("Any text", function (data) {
        console.log( JSON.stringify(data));
      }, function (err) {
        console.log('error from usage plugin: ' + err);
      });
```

**Comment by tolga47** [comment]
Its worked thank you so much :)

**Comment by spec2e** [comment]
👍

**Comment by Vrushabhh** [comment]
Can  you please help me to run this code?
Awaiting early response. Tried many approaches, didn't succeed.
